CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to Ember Blog
+
+New contributors are welcome! Ember Blog is maintained by an all-volunteer team, and we are thankful for your help.
+
+You can support us in a few different ways:
+
+- Suggest an article for [The Ember Times](https://blog.emberjs.com/tags/newsletter.html)
+- Do a write-up of an article for The Ember Times
+- Complete an [issue](https://github.com/ember-learn/ember-blog/issues) with the label "good first issue" or "help wanted"
+- Open an issue to provide feedback
+
+If you have questions or need help with writing, please visit the `#dev-ember-learning` channel on the [Ember Community Discord](https://discordapp.com/invite/zT3asNS).
+
+
+## Setup
+
+<details>
+<summary>Installation</summary>
+
+1. Fork and clone this repo.
+
+    ```bash
+    git clone git@github.com:<your GitHub handle>/ember-blog.git
+    ```
+
+1. Change directory.
+
+    ```bash
+    cd ember-blog
+    ```
+
+</details>
+
+
+<details>
+<summary>Creating a blog post</summary>
+
+1. We use [Markdown](https://www.markdownguide.org/cheat-sheet/) when writing a blog post.
+
+1. The file must be saved in the `source` directory. The file name must begin with `YYYY-MM-DD`.
+
+  ```bash
+  # Create an Ember Times blog post for August 29, 2020
+  touch source/2020-08-29-the-ember-times-issue-162.md
+  ```
```

Comment:
```suggestion
    ```bash
    # Create an Ember Times blog post for August 29, 2020
    touch source/2020-08-29-the-ember-times-issue-162.md
    ```
```